### DISCUSSION TITLE
User/Organization Discussions

### DISCUSSION THREAD

**Discussion by matheus-kirchesch-old** [community/community#14094] [Discussions] [answered]
This forum is amazing, but since its locked to repos, some people that have groups of repos will find this useless, and i guess it would be good to have an org-wide discussion, like we have org-wide projects

**Comment by jricciardi** [comment, marked as answer]
Heya @Kirchesch - you can now enable org-wide discussions. Check out the official documentation here: https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization
